crates/bevy_ui/src/layout/ui_surface.rs

```diff
@@ -463,7 +463,10 @@ mod tests {
         );
     }
 
-    #[allow(unreachable_code)]
+    #[expect(
+        unreachable_code,
+        reason = "Certain pieces of code tested here cause the test to fail if made reachable; see #17231 for progress on fixing this"
+    )]
```

Comment:
I am not proficient enough in `bevy_ui` to know if we should merge these `expect`s that I added to `crates/bevy_ui/src/layout/ui_surface.rs`, or if we should wait until https://github.com/bevyengine/bevy/issues/17231 is fixed.

Reply:
I was informed that there's already a PR for this: https://github.com/bevyengine/bevy/pull/16362